dspace/src/main/docker/README.md

```diff
@@ -130,6 +130,23 @@ docker run -i -t -d -p 80:80 -p 443:443 dspace/dspace-shibboleth
 
 This image can also be rebuilt using the `../docker-compose/docker-compose-shibboleth.yml` script.
 
+## dspace/src/main/docker/dspace-solr/Dockerfile
+
+This Dockerfile builds a Solr image with DSpace Solr configsets included. It
+can be pulled / built following the [docker compose resources](../docker-compose/README.md)
+documentation. Or, to just build and/or run Solr:
+
+```bash
+docker compose build dspacesolr
+docker compose up -d dspacesolr
```

Comment:
I think we should remove or update the `up` line here... as it doesn't work with the instructions in `../docker-compose/README.md` where we start everything with `docker-compose -p d7 up -d`.

I found that this works for me:
```
docker-compose build dspacesolr
# Start all DSpace 7 containers as normal
docker-compose -p d7 up -d
```

Reply:
Done!